docs/01-readme.md

```diff
@@ -120,20 +120,30 @@ Then in gdb set any args and run it:
 
     (gdb) set arg --replace
     (gdb) run
+    
+## Asking questions
+
+#### Stack Overflow
+You can ask questions on [Stack Overflow](http://stackoverflow.com/questions/tagged/awesome-wm).
```

Comment:
however i would personally leave here (and on the website) only reddit, not to spread attention between two platforms